src/attack_flow_builder/src/App.vue

```diff
@@ -151,12 +152,30 @@ export default defineComponent({
 
   },
   async created() {
+    // Detect operating system
```

Comment:
This comment threw me off because I was trying to figure out how the Windows/Linux/Unix OS's get mapped to a settings file. Wouldn't this be a lot simpler if you removed the switch and then used if/else down on lines 168-177?